ban/http/resources.py

```diff
@@ -173,6 +173,32 @@ class Position(VersionnedResource):
     """Manipulate position resources."""
     model = models.Position
 
+    def get_bbox(self, req):
+        bbox = {}
+        req.get_param_as_float('north', store=bbox)
+        req.get_param_as_float('south', store=bbox)
+        req.get_param_as_float('east', store=bbox)
+        req.get_param_as_float('west', store=bbox)
+        if not len(bbox) == 4:
+            return None
+        return bbox
+
+    def get_collection(self, req, resp, **kwargs):
+        qs = super().get_collection(req, resp, **kwargs)
+        bbox = self.get_bbox(req)
+        if bbox:
+            qs = (qs.where(models.Position.center.in_bbox(**bbox))
+                    .group_by(models.Position.pk)
+                    .order_by(models.Position.pk))
+        return qs
+
+    @app.endpoint('/{identifier}/positions')
```

Comment:
Why do we have this new method? Asking "positions" on Position resource does not seem that natural to me ;)